ansible/dashboard-web.yml

```diff
@@ -41,14 +41,23 @@
   roles:
   - { role: software/dashboard/dashboard-db-migrations, tags: [provision, deploy, migrate] }
 
-- name: Tests
+- name: Smoke test dashboard
   hosts: dashboard-web
   tasks:
     - name: flush handlers
       meta: flush_handlers
+    - name: assert app is up
+      uri:
+        url: https://{{ ansible_fqdn }}/
```

Comment:
Isn't this at https://labs.data.gov/dashboard/, not at FQDN?

Reply:
This test is really a host-level check, to make sure the host we just setup is working and configured correctly. labs.data.gov/dashboard would be the end-to-end test since it's going through the load balancer (and some other stuff) and would hit a single random host. An end-to-end test is something I would like to add at some point though.